src/css/flags/media-audio.less

```diff
@@ -8,6 +8,12 @@
             display: table;
         }
     }
+    // This has higher specificity to overwrite caption position when user inactive in playing state
+    &.jw-flag-user-inactive &.jw-state-playing {
```

Comment:
This needs to be this specific. Without the jw-state-playing, the user-inactive caption has the same specificity, which sometimes overwrites (tested on local without the state - works fine, tested on pr builder test - caption moves down when inactive).